themes/demo/partials/site/header.htm

```diff
@@ -16,6 +16,7 @@
                 <li class="{% if this.page.id == 'home' %}active{% endif %}"><a href="{{ 'home'|page }}">Basic concepts</a></li>
                 <li class="{% if this.page.id == 'ajax' %}active{% endif %}"><a href="{{ 'ajax'|page }}">AJAX framework</a></li>
                 <li class="{% if this.page.id == 'plugins' %}active{% endif %}"><a href="{{ 'plugins'|page }}">Plugin components</a></li>
+                <li><a href="{{ backend_url }}">{{ 'backend::lang.dashboard.menu_label'|trans }}</a></li>
```

Comment:
```suggestion
                <li><a href="{{ backend_url }}" target="_blank">Backend</a></li>
```

No need to translate it, nothing else in the demo theme is and it would only apply to the backend locale, which is always going to be english for new installations anyways.

Reply:
You're right.

The French guy (who translate everything).